packages/@coorpacademy-components/package.json

```
@@ -20,6 +20,7 @@
     "build:es": "npm run build:es:babel && npm run build:es:types",
     "build": "concurrently \"npm run build:commonjs\" \"npm run build:es\"",
     "build:watch": "npm run build:es -- -w --verbose",
+    "build:stats": "npx webpack --profile --json > dist/stats.json",
```

Dans ce contexte `npx` n'est pas nécessaire
```suggestion
    "build:stats": "webpack --profile --json > dist/stats.json",
```